docs/RNCamera.md

```diff
@@ -272,6 +272,30 @@ The video stabilization mode used for a video recording. The possible values are
 
 You can read more about each stabilization type here: https://developer.apple.com/documentation/avfoundation/avcapturevideostabilizationmode
 
+### `iOS` `defaultVideoQuality`
+  - `quality`. This option specifies the quality of the video to be taken. The possible values are:
```

Comment:
@listicos This line should be removed seems like it is left over from the old option documentation for `quality`.